Rubberduck.Deployment/Writers/LocalDebugRegistryWriter.cs

```diff
@@ -30,14 +31,12 @@ public string Write(IOrderedEnumerable<RegistryEntry> entries)
                     throw new InvalidOperationException("Unexpected registry entry: " + entry.Key);
                 }
 
-                var value = ReplacePlaceholder(entry.Value, entry.Bitness);
-
-                var key = Registry.CurrentUser.CreateSubKey(entry.Key);
-                if (!(string.IsNullOrWhiteSpace(entry.Name) && string.IsNullOrWhiteSpace(value)))
+                MakeRegistryEntries(entry, Registry.CurrentUser);
```

Comment:
The concern is that this presumes that the powershell script will run in 32-bit mode all the time. That might not be the case because if the script is run in 64-bit Powershell, the keys end up being written to same path twice. 

It would be more appropriate, IMO, to only explicitly write to both 64-bit and 32-bit if the environment is a 64-bit and otherwise use implicit view. (e.g. reverse the condition and statements)

Reply:
Writing twice the same key, yes, now that you mention it that will happen. But also, this powershell is always running within VS build, which is only 32bits (there's no 64bits VS), so I guess we can assume it will always run on 32 (unless Microsoft surprises us with a 64bits VS) (?)
Anyway, let's put it back to the else branch, I also don't like writing to the same key twice.

Reply:
That is true, but I don't like writing code with assumptions that may be one day wrong next year, and especially because it can be very easy to miss the change. 

```
Belts++;
Suspenders++;
```